From 9d3f457d0059b426d3483d0a733f45a2ea8f6dcd Mon Sep 17 00:00:00 2001
From: James McCoy <jamessan@apache.org>
Date: Tue, 24 Mar 2020 23:30:21 +0000
Subject: [PATCH] Remove incorrect include paths from svn_cv_ruby_includes.

Files under ruby/ and ruby/backward/ should be included using "#include
<ruby/...>", if needed, instead of adding those directories to the include
path.

* build/ac-macros/swig.m4
  (SVN_FIND_SWIG): Remove "-I$rbconfig_rubyhdrdir/ruby
   -I$rbconfig_rubyhdrdir/ruby/backward"


git-svn-id: https://svn.apache.org/repos/asf/subversion/trunk@1875602 13f79535-47bb-0310-9956-ffa450edef68
---
 build/ac-macros/swig.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/ac-macros/swig.m4 b/build/ac-macros/swig.m4
index bc2599038f22..c707a8c28253 100644
--- a/build/ac-macros/swig.m4
+++ b/build/ac-macros/swig.m4
@@ -230,7 +230,7 @@ AC_DEFUN(SVN_FIND_SWIG,
     AC_CACHE_CHECK([for Ruby include path], [svn_cv_ruby_includes],[
     if test -d "$rbconfig_rubyhdrdir"; then
       dnl Ruby >=1.9
-      svn_cv_ruby_includes="-I. -I$rbconfig_rubyhdrdir -I$rbconfig_rubyhdrdir/ruby -I$rbconfig_rubyhdrdir/ruby/backward"
+      svn_cv_ruby_includes="-I. -I$rbconfig_rubyhdrdir"
       if test -d "$rbconfig_rubyarchhdrdir"; then
         dnl Ruby >=2.0
         svn_cv_ruby_includes="$svn_cv_ruby_includes -I$rbconfig_rubyarchhdrdir"
